unit-test/stubs/sc_app_stubs.c

```diff
@@ -33,11 +33,11 @@
  */
 CFE_Status_t SC_AppInit(void)
 {
-    UT_GenStub_SetupReturnBuffer(SC_AppInit, CFE_Status_t);
+  UT_GenStub_SetupReturnBuffer(SC_AppInit, CFE_Status_t);
```

Comment:
Looks like whitespace formatting got adjusted all over.  Not sure why CI isn't complaining, but should be complaining that this doesn't match the clang-format rules.